examples/runner.py

```diff
@@ -23,6 +23,7 @@
 ZSEQUENCER_SEND_TXS_INTERVAL: float = 0.01
 ZSEQUENCER_SYNC_INTERVAL: float = 0.01
 ZSEQUENCER_FINALIZATION_TIME_BORDER: int = 120
+ZSEQUENCER_SIGNATURES_AGGREGATION_TIMEOUT = 10
```

Comment:
Let's reduce this default value to 5 seconds

Reply:
Fixed.